client/my-sites/checkout/composite-checkout/payment-method-processors.js

```diff
@@ -123,6 +125,45 @@ export async function stripeCardProcessor(
 	return pending;
 }
 
+export async function ebanxCardProcessor( submitData ) {
+	const paymentMethodToken = await createEbanxToken( 'new_purchase', {
+		country: submitData.countryCode,
+		name: submitData.name,
+		number: submitData.number,
+		cvv: submitData.cvv,
+		'expiration-date': submitData[ 'expiration-date' ],
+	} );
+	const pending = submitEbanxCardTransaction(
+		{
+			...submitData,
+			siteId: select( 'wpcom' )?.getSiteId?.(),
+			deviceId: paymentMethodToken?.deviceId,
+			domainDetails: getDomainDetails( select ),
```

Comment:
Since #44533 this will need to read `getDomainDetails({includeDomainDetails, includeGSuiteDetails})`